backend/src/handlers/RelayBridgeFactory/BridgeDeployed.ts

```diff
@@ -19,6 +19,7 @@ export default async function ({
       createdAt: event.block.timestamp,
       createdAtBlock: event.block.number,
       transferNonce: BigInt(0),
+      updatedAt: event.block.timestamp,
```

Comment:
here we are indexing events from chains, so records will never be updated once they are written on chain.

Reply:
ok, the camel case will be automatically turn into snake case for the column names. I added the indexes on both columns. If the data is reset entirely then the best approach may be to delete and recreate. The data in the db is just an index of existing onchain events so no problem about droping it